collections/user.js

```diff
@@ -60,6 +60,109 @@ Meteor.methods({
         }, {
             $set: user
         });
+    },
+
+    searchInterestedGuests: function(topics, limit) {
+        for (var i in topics) {
+            if (topics.hasOwnProperty(i)) {
+                topics[i] = new RegExp('^' + topics[i], 'i');
+            }
+        }
+
+        /**
+         * We find in users collection if some of the topics
+         * are present in their interests or skills
+         * @type {Array}
+         */
+        var users = Meteor.users.find({
+            $or: [{
+                'profile.interests': {
+                    $in: topics
+                }
+            }, {
+                'profile.skills': {
+                    $in: topics
+                }
+            }]
+        }, {
+            '_id': 0,
+            'emails': 1,
+            'profile.skills': 1,
+            'profile.interests': 1,
+            'visitedRooms': 1
+        }).fetch();
+
+
+        for (var user in users) {
+            if (users.hasOwnProperty(user)) {
```

Comment:
¿Por qué se hace este chequeo? la existencia de la key `user` ya debería estar garantizada por el `for`

Esto debería ser algo así:

```
for (var i in users) {
    var user = users[i];
    ...
}
```